File: OldSnacks/OldSnackjs/GenerateRandom.js
const prompt = require("prompt-sync")();

const generateRandom = function(numberOne){

	let randomGuess = Math.floor(Math.random() *10)
	let randomGuessTwo = Math.floor(Math.random() *10)

	let result = 0
	let check = 0

	for(let count = 1; count <= 10; count++){
		
		 randomGuess = Math.floor(Math.random() *10)
		 randomGuessTwo = Math.floor(Math.random() *10)

		if(randomGuess > randomGuessTwo ){
		console.log(`${randomGuess} - ${randomGuessTwo}`)
		result = randomGuess - randomGuessTwo
		}

		let user = Number(prompt("Enter correct answer: "))	
		check++

		if (user == result && check == 2){
		console.log("correct answer")
		
		
		} else if (user < result || user > result && check == 2){
			console.log("Try again")
		}

	}
	console.log("your final Score is: ", check)

}
generateRandom(Math.random(20, 10))

File: OldSnacks/OldSnacksjava/GenerateRandom.java
import java.util.Random;
import java.util.Scanner;

public class GenerateRandom{

	public static void main(String[] args){

	Random myRandom = new Random();
	Scanner sc = new Scanner(System.in);

	int check = 0;
	int result = 0;
	int correct = 0;
	for(int count = 1; count <= 10; count++){

		int randomGuess = myRandom.nextInt(10);
		int randomGuessTwo = myRandom.nextInt(10);

		if(randomGuess > randomGuessTwo){
		System.out.printf("%d - %d%n", randomGuess, randomGuessTwo);	

			System.out.print("Enter guess: ");
			int input = sc.nextInt(); 
			result = randomGuess - randomGuessTwo;
			check++;
			
			if(input == result && check == 2 ){
				System.out.print("correct guess");
				correct++;
			}			
			else if(input < result || input > result && check == 2){
				System.out.print(" Try again ");
			}
		}
		
	}
	System.out.printf(" Your result is %d%n" , correct);

}

}